app/services/query-builder.js

```diff
@@ -76,7 +76,7 @@ export const associationsQuery = ({ index, page, params, size }) => {
     } else if (params.status === 'Verlopen') {
       addFilter(
         ':query:recognitions.validityPeriod',
-        `(NOT ((recognitions.validityPeriod.startTime:<=${today}) AND (recognitions.validityPeriod.endTime:>=${today})))`,
+        `recognitions.validityPeriod.endTime:<${today}`,
```

Comment:
Without this change it would also return the "future" recognitions when filtering on "Verlopen". It's still not perfect, selecting both "Verlopen" and "Erkend" options at the same time returns all "Verlopen", "Erkend" and "future" statuses, but I'm not sure how to fix that.

Does it actually need to be a checkbox filter? Do we want to allow users to select multiple statuses?

Reply:
@Windvis With this change I get also the active recognitions when selecting "verlopen". 
When the association only has one recognition this is working but with multiple recognitions it's not working..

<img width="2197" alt="image" src="https://github.com/user-attachments/assets/1f78e624-d33a-457a-865f-9f8455f7a2c6">
<img width="2547" alt="image" src="https://github.com/user-attachments/assets/e9720806-4fac-4392-8988-ba7da0d83037">

Reply:
I also think we should change this from a checkbox to a radio button or a dropdown. To only select one recognition status at a time. We should ask this to Sofie.

Reply:
@Andresdev02 But it does have a recognition that's expired, maybe that's why? I don't know enough about the search setup, what action should I take here?

Reply:
@Windvis Yes that is true but it should only have the following status:
If association has only expired recognitions -> expired 
If it has an active recognitions and also expired recognitions -> recognized
If it has a future recognition and also an active recognitions -> recognized
Only future -> nothing
Only active -> recognized

If both are selected in the filter we should show every associations with a recognition except the recognitions with only future recognitions.